bin/travis/checkstrings.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
+OUTPUT=$(sh ./bin/updatestrings.sh --dry-run)
+ADDED_MESSAGES=$(echo "$OUTPUT" | awk '/Added Messages/ { print $3 }')
+DELETED_MESSAGES=$(echo "$OUTPUT" | awk '/Deleted Messages/ { print $3 }')
+
+RETURN=0
+
+if test "$ADDED_MESSAGES" != 0; then
```

Comment:
usually this is written with `[]` instead of `test` and also to avoid any ambiguity, I would compare against `"0"` not `0`, so something like:

```sh
if [ "$ADDED_MESSAGES" != "0" ] ; then
```